docs/src/infra/protocols/telnet.md

```diff
@@ -1,6 +1,144 @@
 ---
-authors: ShutdownRepo
+authors: ShutdownRepo, Tednoob17
 ---
 
 # 🛠️ Telnet
 
+## Theory
+Telnet (teletype network) is a  network protocol used to gain access to virtual terminal in local or in remote systems . He provide  bidirectional text-based communication .
+
+----
+### Common Telnet Commands
+
+| Command   | Description                                                    |
+| --------- | -------------------------------------------------------------- |
+| `open`    | Connects to a specified local/remote host                      |
+| `close`   | Closes the current connection                                  |
+| `quit`    | Exits telnet                                                   |
+| `status`  | Shows the current status of the telnet client                  |
+| `z`       | Suspends telnet (on Unix/Linux systems)                        |
+| `set`     | Sets Telnet options (like terminal type)                       |
+| `unset`   | Unsets Telnet options                                          |
+| `display` | Displays current settings of Telnet options                    |
+| `send`    | Sends special characters or sequences (like break)             |
+| `mode`    | Sets the mode of operation (e.g., line by line or character)   |
+| `logout`  | Logs out from the remote system (not available on all systems) |
+
+
+## Practice
+### Enumeration
+#### Banner Grabbing
+To initiate a connection with telnet server and get any information about the target .
+
+
+> [!TIP]
+> The $TARGET_PORT is optional, but the default port is 23
+
+
+#### CLI tool
+::: tabs
+
+=== Unix-like
+
+```bash
+#Using Netcat
+nc -nv $TARGET_IP $TARGET_PORT
+#Using Telnet
+telnet $TARGET_IP $TARGET_PORT
+#Using Shodan-cli
+shodan stream --ports 23,1023,2323 --datadir telnet-data/ --limit 10000
+#Using nmap
+nmap -p  $TARGET_PORT -sVC  --script "*telnet* and safe" $TARGET_IP
+```
+
+
+=== Windows
+
+```powershell
+nc.exe -nv $TARGET_IP $TARGET_PORT
+telnet $TARGET_IP $TARGET_PORT
+nmap -p  $TARGET_PORT   --script telnet-ntlm-info.nse $TARGET_IP
+```
+:::
+
+#### Automated tools
+The following commands is typed in msfconsole (The Cli for Metasploit Framework)
+
+::: tabs
+
+=== Metasploit 
+
+```
+msf > use auxiliary/scanner/telnet/telnet_version
+msf > set rhosts $TARGET_IP 
+msf > set rport $TARGET_PORT
+msf > set threads 5
+msf > exploit
+```
+:::
+
+
+### Attacks 
+
+#### Passwordless Authentication
+
+Telnet allows users to connect to a server without needing a specific identity by utilizing a passwordless login feature. This method is commonly employed for accessing or downloading public files.
+
+To connect without a password, you would use the following command:
+
+```bash
+telnet $TARGET_IP
+
+#provide username
+#but not provide any password
```

Comment:
Can you add a space after the # ?
Also make it one sentence please : "# provide username, without password"